setup.py

```diff
@@ -32,10 +34,9 @@
 DOWNLOAD_URL = 'https://github.com/nipy/PySurfer'
 VERSION = version
 
-def check_dependencies():
 
-    needed_deps = ["IPython",
```

Comment:
I don't know why we had `IPython` listed as a dependency... @mwaskom looks like this was your change any idea?